src/services/twilio.ts

```diff
@@ -103,7 +103,7 @@ export const TwilioClient = (): IPhoneProviderService => {
 
       const phoneMetadata: PhoneMetadata = {
         carrier: {
-          error_code: result.carrier.error_code,
+          error_code: String(result.carrier.error_code),
```

Comment:
There was an edge-case that @dolcalmi had identified when using `String` so we started casting to string like this in the codebase now

```suggestion
          error_code: `${result.carrier.error_code}`,
```

Reply:
do you have more context on this edge case @dolcalmi @vindard ?